packages/typography/src/Typography.tsx

```diff
@@ -14,14 +14,12 @@ const sharedStyles = css`
 `;
```

Comment:
(can't comment on line 10, thanks GitHub)

I realized that we're setting display block on just about everything. Headers are already display block by default, and the only tag that block being set on that isn't already is the `<small>` tag.

Reply:
Kk fixed!